less/kikar.less

```diff
@@ -25,4 +25,72 @@
 #loading-statuses-symbol{
   text-align: center;
   display: none;
-}
\ No newline at end of file
+}
+
+
+/*  Styling for responsive embedding by http://embedresponsively.com/*/
+
+.embed-container {
+    position: relative;
+    padding-bottom: 56.25%;
+    height: 0;
+    overflow: hidden;
+    max-width: 100%;
+}
+
+.embed-container iframe, .embed-container object, .embed-container embed {
```

Comment:
missing the point of using less should be part of the class above someClass:{

variousettings:11px;
iframe, object, embed {
anothe setting:12px
}
}